community/sustainability_effort.md

```diff
@@ -11,6 +11,7 @@ Help is needed for:
 * [triaging](../docs/CONTRIBUTING.md#triaging-bugs) new bugs by prioritizing them with `P0`, `P1`, `P2`, and `P3` labels
 * responding to questions in [Github Discussions](https://github.com/argoproj/argo-workflows/discussions)
 * responding to questions in [CNCF Slack](https://argoproj.github.io/community/join-slack) in the `#argo-workflows` and `#argo-wf-contributors` channels 
+* performing releases
```

Comment:
"performing" is a bit of an ambiguous word and the construction feels off when "releasing" is itself a verb (similar to triaging above)... I'm not totally sure what to replace this with though 🤔

a few different suggestions below, roughly in order of my preference (up to you though):

```suggestion
* releasing new versions
```

```suggestion
* creating releases
```

```suggestion
* publishing releases
```

```suggestion
* publishing new versions
```

I definitely had a few others in my head that I forgot 😅
The first still sounds most apt, because it is still a bit amorphous to allow for multiple kinds of tasks within the definition of "release", but is clearer than "performing"